README.md

```diff
@@ -28,9 +28,10 @@ All scripts are run built with python3, but should be able to be run with python
 4. Based on config keys in `analyze` markdown, csv, and json will be generated for the aggregated sims. These will output separate files for Composite, Single Target, or Dungeon sims. You can find all output files in the `results/` folder in any sim folder.
 
 #### Use local simc
-1. To run with a local simc you have to options:
+1. To run with a local simc you have three options:
 - use the simc you have on your Path ( nothing to setup here )
 - use a simc located in a separated folder, create a `local_secrets.py` inside the root directory and set `simc_path = '{"nightly": "path/to/executable", "rework": "another/path"}'`
+- add the `--auto_download` parameter to download the latest simc nightly build into the `auto_download/` folder. This will override any choice in `config.yml` with the path to the newly downloaded version.
```

Comment:
Maybe its worth mentioning here that this --auto_download option is only possible within windows based systems ;). Because i don't think "C:\.....7zip.exe" is a good way to try unpack stuff under linux ;).

Reply:
Good point for sure, if needed i could add support for linux/osx. I just wasn't sure of the usage.

Reply:
I will take a look at this :) i only use linux so maybe i can help here :)

Reply:
Hmm i think for the moment we can't support linux here only osx would be possible because simc is not build for linux seperate :). And osx i currently don't have a installation available :).

Reply:
Yeah that would be perfect i think thats the best for the moment :), if i find a mac somewhere around me i will try to get it working there as well. Thanks!

Reply:
Supporting different platforms not only requires adjusting 7z, but also the architecture name of the simc nightly file.

Might be nice to support macos, but you could also just hard-check for os windows in python, and throw a error if someone uses it on a unsupported platform.

Reply:
@scamille like i mentioned at the other place its 7z is not the way to go for mac and linux. I can't test all this because i only have linux at home ;). Maybe create a issue for supporting macos, so we don't make the conversation here any longer :). For the moment i think the notice is okay, and the support for macos can be implemented in a seperate pull request.